platform_plugin_ontask/static/html/ontask.html

```diff
@@ -1,3 +1,39 @@
 <div class="ontask-instructor-wrapper">
-    <iframe class="ontask-instructor-iframe" src="{{ONTASK_URL}}" frameborder="0"></iframe>
+  {% if workflow_id %}
+  <div class="upload-dataframe">
+    <button
+      id="upload-dataframe"
+      data-course-id="{{ course_id }}"
+      class="btn btn-primary upload-dataframe-button"
+    >
+      Load data
+    </button>
+    <span id="upload-dataframe-message" class="success-message"></span>
+    <iframe
+      class="ontask-instructor-iframe"
+      src="{{ ontask_url }}/action/{{ workflow_id }}/index"
+      frameborder="0"
+    ></iframe>
+  </div>
+  {% else %}
+  <div class="create-workflow-wrapper">
+    <h2 class="hd hd-2">Welcome to OnTask Open edX Connection</h2>
+    <button
+      id="create-workflow"
+      data-course-id="{{ course_id }}"
+      class="btn btn-primary create-workflow-button"
+    >
+      Create workflow
+    </button>
+    <span
+      id="create-workflow-error-message"
+      class="error-message hidden"
+    ></span>
+    <h3 class="hd hd-3">There is no workflow ID associated with the course.</h3>
+    <p>
+      Create a new one or if you already have one, configure it in the Advanced
+      Settings of the course.
```

Comment:
can we specify how to configure it? Something like: add these keys to the other course settings dictionary with the correct values?

Reply:
Sure, here I added a more detailed message: https://github.com/eduNEXT/platform-plugin-ontask/commit/8c50eb263e20a0039a3150352917bc717e14dd8f